testnet/stacks-node/src/neon_node.rs

```diff
@@ -287,33 +493,66 @@ fn spawn_peer(
                 );
                 100
             } else {
-                poll_timeout
+                cmp::min(poll_timeout, config.node.microblock_frequency)
+            };
+
+            // Mine microblocks.
+            // NOTE: this has to go *here* because control over who can refresh the unconfirmed
```

Comment:
Once the Clarity database refactor is finished, can the microblock mining occur in the miner/relayer thread? If so, can you add a TODO to this note to update this path?

Reply:
Yes, I think it can be moved to the relayer once it's safe to have multiple threads writing to the DB.

Reply:
Addressed in ee12847